fungible-asset-voting/move/sources/voting.move

```diff
@@ -0,0 +1,392 @@
+module voting_app_addr::voting {
+    use std::bcs;
+    use std::signer;
+    use std::string::String;
+    use std::vector;
+    use aptos_framework::object::{Self, ExtendRef, Object};
+    use aptos_framework::timestamp;
+    use aptos_std::string_utils;
+    use aptos_framework::fungible_asset::{Self, Metadata, FungibleStore};
+    use aptos_framework::primary_fungible_store;
+
+    // Error Codes
+    const ERR_LIVE_PROPOSAL_ALREADY_EXISTS: u64 = 1;
```

Comment:
we need to have doc string for each error code, this will show up in the wallet when the error is triggered to help users better understand the error.

can refer to [here](https://github.com/aptos-labs/create-aptos-dapp/blob/main/templates/token-staking-dapp-template/move/sources/stake_pool.move#L16)